packages/commonwealth/server/migrations/20240620220459-update-alchemy-chain-nodes.js

```diff
@@ -17,9 +17,9 @@ module.exports = {
 
     await queryInterface.sequelize.query(`
       UPDATE "ChainNodes"
-      SET url = regexp_replace(url, '\\/([^\\/]*)$', '${publicKey}'),
-          alt_wallet_url = regexp_replace(url, '\\/([^\\/]*)$', '${publicKey}'),
-          private_url = regexp_replace(url, '\\/([^\\/]*)$', '${privateKey}')
+      SET url = regexp_replace(url, '\\/([^\\/]*)$', '/${publicKey}'),
+          alt_wallet_url = regexp_replace(url, '\\/([^\\/]*)$', '/${publicKey}'),
+          private_url = regexp_replace(url, '\\/([^\\/]*)$', '/${privateKey}')
```

Comment:
Prefer that migrations are immutable, but since this is fairly new I guess it's no biggy

Reply:
Yea usually I would agree but this hasn't deployed, it's a quick follow-up, and ICs who run `db-all` won't notice. Tryna sneak it in :p